### ISSUE TITLE
[Feature request] Random toggle for autowallpaper

### ISSUE THREAD

**Issue by Le0xFF** [ammargitham/WallFlow#91]
Hello and big thanks again for this application!

As you suggested, I opened a new feature request.
With version v2.5.0 "Auto wallpaper will choose next wallpaper in sequence instead of random to prevent re-using previous wallpapers".

The request is to have a toggle that enable/disable the randomization of wallpaper, as I guess there are people like me that don't mind having the same wallpaper applied twice.
Basically asking for number 1 of issue #57.

Thank you so much for your time and sorry for my bad english!
